tools/lintapidiff.ml

```diff
@@ -126,13 +126,14 @@ module Ast = struct
       match ty.pmty_desc with
       | Pmty_signature lst -> List.fold_left self map lst
       | Pmty_functor ({txt;_}, _, m) ->
-          let path = Path.Papply(path, Path.Pident (Ident.create txt)) in
+          let new_name = Ident.create_local txt in
```

Comment:
The identifier should be `Ident.create_persistent` since we want identifier to share paths between different versions of the standard library.